### PR TITLE
PVP-284 Add localization support

### PR DESCRIPTION
**Pull request by klaidoshka**
# Changes
- Discarded some of the unused UI code
- Added locale setting
- Supporting EN (Default/Unset), LT locales
- String resources must be used for locale to work as intended
- Translated:
  - Screens
  - Popups
  - Notifications
  - Forms
- Not translated:
  - Icons contentDescription, I believe this is redundant. I may be wrong, maybe fully lithuanian accessibility tools for blind people exist and are used in lithuanian. Let me know whether I should spend time for this.
- Fixed goal creation form bug when creating Walking goal with kilometers distance type
- Adjusted snackbar text position to make it fit into bounds and be in the center

### CODE REVIEW

File: app/src/main/res/values-lt/strings.xml [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+    <string name="friends_sort_type_tasks">Užduotys</string>
+    <string name="friends_success_already_received_request">Tiek jūs, tiek jūsų draugas kvietėtės vienas kitą. Užklausos priimtos</string>
+    <string name="friends_success_remove">Draugas sėkmingai panaikintas</string>
+    <string name="friends_success_request_accepted">Draugo užklausa priimta</string>
+    <string name="friends_success_request_canceled">Draugavimo užklausa atšaukta</string>
+    <string name="friends_success_request_denied">Draugo užklausa atmesta</string>
+    <string name="friends_success_sent">Draugavimo užklausa išsiųsta</string>
+    <string name="friend_7_days_activity">7 dienų aktyvumas</string>
+    <string name="friend_button_remove">Panaikinti draugą</string>
+    <string name="friend_button_remove_confirmation_description">Jeigu draugas bus panaikintas, turėsite jį pakviesti iš naujo į draugus.</string>
```
Comment (vytjar) [inline app/src/main/res/values-lt/strings.xml:124]: Maybe just "Jeigu draugas bus panaikintas, turėsite jį pakviesti iš naujo" would suffice? The last "į draugus" sounds a bit odd.

File: app/src/main/res/values-lt/strings.xml [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+    <string name="friend_friends_since">Draugai nuo - %1$s</string>
+    <string name="friend_goals_completed">%1$s siekių pabaigta</string>
+    <string name="friend_mutual_friends">Bendri draugai</string>
+    <string name="friend_mutual_friends_error">Neturite bendrų draugų</string>
+    <string name="friend_other_information">Kita informacija</string>
+    <string name="friend_sport_tasks_completed">%1$s sportinių užsiėmimų pabaigta</string>
+    <string name="friend_steps_made">%1$s žingsnių</string>
+    <string name="global_experience">Patirtis</string>
+    <string name="global_level">Lygis</string>
+    <string name="goals_card_average_monthly_steps">Jūsų vidutiniškai atliekami žingsniai mėnesyje %1$s</string>
```
Comment (vytjar) [inline app/src/main/res/values-lt/strings.xml:137]: Maybe "Vidutinis nueitų žingsnių kiekis per mėnesį"? Same with weekly.

File: app/src/main/res/values-lt/strings.xml [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+    <string name="settings_setting_google_calendar_synchronize_success">\"Google\" Kalendorius sėkmingai sinchronizuotas</string>
+    <string name="settings_setting_google_calendar_title">\"Google\" Kalendorius</string>
+    <string name="settings_setting_health_connect_button_configure">Konfigūruoti</string>
+    <string name="settings_setting_health_connect_description">Skirdami tam tikrus \"Health Connect\" leidimus padedate mums pateikti daugiau puikių funkcionalumų jums. Skirtus leidimus bet kada galite išjungti ateidami čia. \"Health Connect\" mums leidžia pasinaudoti kitų programėlių surinktais duomenimis apie jūsų aktyvumą.</string>
+    <string name="settings_setting_health_connect_title">\"Health Connect\" įrankis</string>
+    <string name="settings_setting_hydration_reminder_description">Pasirinkite ar norite įjungti vandens išgėrimo priminimus</string>
+    <string name="settings_setting_hydration_reminder_title">Vandens Išgėrimo Priminimas</string>
```
Comment (vytjar) [inline app/src/main/res/values-lt/strings.xml:283]: Sounds a bit odd. Maybe "Vandens suvartojimo" ar "Vandens vartojimo" would be better? Unsure though.

File: app/src/main/res/values-lt/strings.xml [hunk 4]
```diff
@@ -0,0 +1,328 @@
...
+    <string name="decorations_error_own">Jūs jau turite šią dekoraciją</string>
+    <string name="decorations_none_to_apply">Nėra dekoracijų, kurias galėtumėte užsidėti</string>
+    <string name="decorations_none_to_purchase">Nėra dekoracijų, kurias galėtumėte nusipirkti</string>
+    <string name="decorations_owned_title">Inventorius</string>
+    <string name="decorations_points">Jūsų taškai %1$s</string>
+    <string name="decorations_purchase_dialog_decoration">Dekoracija</string>
+    <string name="decorations_purchase_dialog_price">Kaina</string>
+    <string name="decorations_purchase_dialog_title">Patvirtinkite pirkimą</string>
+    <string name="decorations_store_title">Parduotuvė</string>
+    <string name="decorations_success">Pavyko atlikti operaciją</string>
```
Comment (vytjar) [inline app/src/main/res/values-lt/strings.xml:73]: "Operacija atlikta sėkmingai" sounds a bit nicer to me?

**Review by vytjar — APPROVED** [review]

File: app/src/main/res/values-lt/strings.xml (hunk 3 above)
Comment (klaidoshka) [reply app/src/main/res/values-lt/strings.xml:283]: Decided to stay for `Gėrimo`

**Pull request merged**